moveit_experimental/moveit_jog_arm/srv/ChangeControlDimensions.srv

```diff
@@ -0,0 +1,8 @@
+# Turn on/off jogging along these dimensions.
+# Order matters: [translation-x, translation-y, translation-z, rotation-x, rotation-y, rotation-z]
+# Give 'true' to enable jogging in the direction, 'false' to disable
+# E.g. [true, false, false, true, false, false] would enable jogging in the EEF x translation and roll
+# An empty array enables all control dimensions
+bool[] dimensions_to_control
```

Comment:
I'm only an observer here, so take this as a comment: a `bool` array has really poor semantics. Interpretation of the elements is entirely dependent on information that is not captured by this message, and additionally it is very easy to make mistakes indexing into arrays (off-by-ones are very common).

As there are only 6 dimensions, and they all have names, my suggestion would be to use fields with proper names. No mistakes indexing into arrays, clear semantics and immediately clear what the fields represent.

Reply:
That sounds reasonable to me

Reply:
Agree. Changed, tested, and committed.